frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.ts

```diff
@@ -50,6 +53,16 @@ export class ExperimentListComponent implements OnInit, OnDestroy, AfterViewInit
   @ViewChild('searchInput') searchInput: ElementRef;
 
   @ViewChild(MatSort, { static: true }) sort: MatSort;
+  searchControl = new FormControl();
+
+  get filteredStatusOptions(): string[] {
+    if (this.searchValue !== undefined) {
```

Comment:
we really only want searchValue to specifically be a string, so I think we'd be better to be explicit `if (typeof this.searchValue === 'string')`.